invictusd0a/inn-api
# #165369082 Add authentication test cases

### **What does this PR do?**

This PR adds a tests for the user authentication.

### **How to run tests**
Run unit tests.
`python manage.py test -v2` or `coverage run manage.py test`

Run coverage report
`coverage report`

Run for htmlcov to see missing lines
`coverage html`


### **Relevant user stories**

[#165369082](https://www.pivotaltracker.com/story/show/165369082)
